SettingsPage/SettingsPage.pt-BR.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Delay": "Atraso",
+  "EditSources": "Editar fontes...",
+  "General": "Geral",
+  "InstallPlugin": "Instalar plugin...",
+  "LoadedPlugins": "Plugins carregados",
+  "Nicknames": "Apelidos",
+  "NoPluginSensorsHelpText": "Nenhum sensor do plugin detectado. Certifique-se de que o plugin está habilitado na janela de fontes de sensores e que o aplicativo foi reiniciado.",
+  "NoPlugins": "Nenhum plugin carregado",
+  "NotifyForUpdate": "Notificar sobre atualizações na inicialização",
+  "OriginalName": "Nome original: {0}",
+  "PluginSensors": "Sensores dos plugins",
```

Comment:
```suggestion
  "PluginSensors": "sensores dos plugins",
```

Reply:
Not sure why but the original is not titled
https://github.com/Rem0o/FanControl.i18n/blob/main/SettingsPage/SettingsPage.json#L12